### PR TITLE
chore: moved TrustDefender into this repository

### PR DESCRIPTION
**Pull request by jarrodrobins**
Previously, TrustDefender was part of a separate repository. This moves that SDK into here.

### CODE REVIEW

**Review by haitian-jin-awx — APPROVED** [review]

File: Podfile [hunk 1]
```diff
@@ -7,6 +7,17 @@ ENV['SWIFT_VERSION'] = '5'
 
 workspace 'Airwallex.xcworkspace'
 
+def shared_profiling_pods
+pod 'AirwallexBehavioralBiometrics', path: 'TrustDefender'
```
Comment (JinW-Airwallex) [inline Podfile:11]: nitpicking: missing space in front

File: TrustDefender/AirwallexBehavioralBiometrics.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name					= 'AirwallexBehavioralBiometrics'
+  s.version					= '6.1.55'
+  s.summary					= 'TrustDefender framework'
+  s.homepage				= "https://www.airwallex.com/docs/#overview"
+  s.license					= { :type => 'MIT', :file => 'LICENSE' }
+  s.author					= { 'Airwallex' => 'pa_mobile_sdk@airwallex.com' }
+  s.source					= { :git => 'https://github.com/ilabsvictor/TrustDefender.git', :tag => s.version.to_s }
```
Comment (JinW-Airwallex) [inline TrustDefender/AirwallexBehavioralBiometrics.podspec:8]: We should update these files accordingly. They also have weird format.

File: Podfile (hunk 1 above)
Comment (jarrodrobins) [reply Podfile:11]: Whoops, that's an oversight. Not a nitpick at all!

Comment (jarrodrobins) [reply Podfile:11]: Whoops, that's an oversight. Not a nitpick at all!

**Pull request merged**